src/devices/sound/sn76496.cpp

```diff
@@ -126,7 +126,7 @@
   * NCR8496's output is inverted, PSSJ-3's output is not.
 
   10/12/2019: Michael Zapf
-  * READY line handling by own emu_timer, not depending on sound_stream_update
+  * READY line handling by own emu_timer, not depending on sound_stream_update_legacy
```

Comment:
This comment is out-of-sync with the source - it doesn’t implement `sound_stream_update_legacy`.

Reply:
Search/replace, thanks.